test/RoaveTest/SecurityAdvisories/VersionConstraintTest.php

```diff
@@ -375,8 +375,8 @@ public function mergeableRangesProvider() : array
             ['>1,<2.10', '>1,<2.100', false, true],
             ['>1.0,<2', '>1,<2', true, true],
             ['>1,<2.0', '>1,<2', true, true],
-            ['>1.0.0,<2', '>1,<2', true, true],
```

Comment:
This diff should be an addition, not a change. The necessity for a change comes from how `Version` is no longer normalised at instantiation, but later on.

If `1` and `1.0.0` are the same version (literally `$version1 == $version2`), then this works again